### PR TITLE
[Tile - Usage] Website Doc 

### PR DESCRIPTION
**Pull request by Kritvi-bhatia17**
Updated the Usage tab docs on the website for the Tile component.

#### Changelog

**New**
- Add 'the experimental tile' link in the inline notification 

**Changed**
- Just a few minor updates to the copy to ensure consistency
      - Tile variants -> Variants

### CODE REVIEW

File: src/pages/components/tile/usage.mdx [hunk 1]
```diff
@@ -29,9 +29,9 @@ in their products moving forward.
 
 <AnchorLink>Live demo</AnchorLink>
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Tile variants</AnchorLink>
+<AnchorLink>Variants</AnchorLink>
 <AnchorLink>Formatting</AnchorLink>
-<AnchorLink>Universal behaviors</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
```
Comment (Kritvi-bhatia17) [inline src/pages/components/tile/usage.mdx:34]:
```suggestion
<AnchorLink>Universal behaviors</AnchorLink>
```

File: src/pages/components/tile/usage.mdx [hunk 2]
```diff
@@ -411,7 +413,7 @@ version.
 </Column>
 </Row>
 
-## Universal behaviors
+## Behaviors
```
Comment (Kritvi-bhatia17) [inline src/pages/components/tile/usage.mdx:416]:
```suggestion
## Universal behaviors
```

**Review by alina-jacob — APPROVED** [review]
Looks good Kritvi! 🚀

**Review by thyhmdo — APPROVED** [review]
I think there are two ways to do this. Our current usage doc doesn't have variants as H2, but H3 instead. But there are other docs following this format, probably the old way. It should be fine for now though until we are more consistent.

**Comment by Kritvi-bhatia17** [comment]
> H3

I've already tried to fix the `variant's inconsistency` in [this PR](https://github.com/carbon-design-system/carbon-website/pull/3969#issue-2183476456)

**Pull request merged**